addons/hide-project-stats/userscript.js

```diff
@@ -2,22 +2,25 @@ export default async function ({ addon, global, console, msg, safeMsg: m }) {
   let loves = { name: "love" };
   let favorites = { name: "favorite" };
   let username = await addon.auth.fetchUsername();
+  let visitingOwnProject = username !== null && document.querySelector(".button.action-button.report-button") === null;
```

Comment:
Project might have loaded yet. So report button might not exist yet for a project made by someone else.